ui/puzzle/css/_tools.scss

```diff
@@ -130,7 +130,6 @@
 
   position: relative;
   height: $col3-uniboard-controls;
-  margin-top: #{-$puzzle-block-gap};
```

Comment:
Removing this instead of fixing it because the current placement of the puzzle controls looks fine.